lib/moab/storage_object_validator.rb

```diff
@@ -92,14 +95,32 @@ def check_correctly_formed_moabs
         errors.concat check_sub_dirs(version_sub_dirs, version_dir, EXPECTED_VERSION_SUB_DIRS)
         after_result_size = errors.size
```

Comment:
don't even compute "after_result_size" because you can't use it on line 99, and you're not using it on line 98 either ?